exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -136,26 +136,28 @@ export default ExpFrameBaseComponent.extend({
   // Represent the sorted cards in a human-readable format for storage in the database
   responses: Ember.computed(function() {
       // Final data should be returned as {
-      //    cardSort1: object {categoryName: [cardIdentifiers] }
-      //    cardSort2: object {categoryName: [cardIdentifiers] }
+      //    cardSort1: object {cardId: categoryId,...}
+      //    cardSort2: object {cardId: categoryId,...}
       // }
       let responses = {};
       let cardSortResponse = this.get('cardSortResponse');
       if (cardSortResponse) {
           responses['cardSort1'] = {};
-          for (let category of cardSortResponse) {
-              let name = category.name.split('.').pop();
-              responses['cardSort1'][name] = category.cards.map((cardItem) => cardItem.id);
+          for (var i=0; i < cardSortResponse.length; i++) {
+              for (let card of cardSortResponse[i].cards) {
+                  responses['cardSort1'][card.id] = i + 1;
+              }
           }
       }
       if (this.get('page') === 'cardSort2') {
           cardSortResponse = this.get('buckets2');
           responses['cardSort2'] = {};
           // Assumption: this unpacks a list of { categories: {name: name, cards: [cards]} } objects
           for (let categorySet of cardSortResponse) {
-              for (let category of categorySet.categories) {
-                  let name = category.name.split('.').pop();
-                  responses['cardSort2'][name] = category.cards.map((cardItem) => cardItem.id);
+              for (var j=0; j < categorySet.categories.length; j++) {
```

Comment:
Does this extra level of iteration indicate a change in the data structure we're operating on? Can you please verify that the source code comments are accurate and up-to-date?

Is the final output of this computed a category ID (as suggested in line 139), or a numeric value corresponding to the category?